pilot/pkg/bootstrap/server.go

```diff
@@ -918,6 +906,10 @@ func (s *Server) initRegistryEventHandlers() {
 				Resource().GroupVersionKind() {
 				continue
 			}
+			if schema.Resource().GroupVersionKind() == collections.IstioNetworkingV1Alpha3Workloadgroups.
```

Comment:
Do you mean workloadEntry or workloadGroup?

Reply:
It is workload group here

Reply:
wlg currently can not influence any xds